src/xcoordclass.js

```diff
@@ -129,6 +133,10 @@ XCoord.prototype = {
         // as soon as it get as close as allowed to it's right neighbour
 
     var rightBoundary = this.getRightMostNoDisturbPosition(v);
+    if (rightBoundary === Infinity || rightBoundary === -Infinity) {
```

Comment:
Isn't it the code written twice? 😃

Reply:
...with one variable

Reply:
`const returnIfOutside = boundary => (boundary === (Infinity || boundary === -Infinity) && return 0)`

Reply:
At the very least we could probably use `isFinite` to shorten this a little.

`if (Number.isFinite(rightBoundary)) return 0;`

Reply:
Just a question about the coding habits - does somebody prefer to use where it's possible such forms?: `(rightBoundary === Infinity || rightBoundary === -Infinity) && return 0`

Reply:
...or even `Number.isFinite(rightBoundary) && return 0`?

Reply:
simplified the check using `Number.isFinite` and moved it to a function. As to the notation, personally, I prefer using `if` over a short form, it seems a bit more readable to me. Let's see what other reviewers think?

Reply:
I generally prefer the if format it is more obvious when used outside of React.